android/Staccato_AN/app/src/main/java/com/on/staccato/presentation/staccatoupdate/viewmodel/StaccatoUpdateViewModel.kt

```diff
@@ -273,13 +276,41 @@ class StaccatoUpdateViewModel
             _currentPhotos.value = currentPhotos.value?.updateOrAppendPhoto(updatedPhoto)
         }
 
-        private fun handleError() {
+        private fun handleServerError(
+            status: Status,
+            message: String,
+        ) {
+            _isPosting.value = false
+            _warningMessage.postValue(message)
+        }
+
+        private fun handleException(
+            e: Throwable = IllegalArgumentException(),
+            errorMessage: String = "필수 값을 모두 입력해 주세요",
+        ) {
```

Comment:
상수화를 해주시면 좋을 것 같아요!
그리고 현재 사용되고 있는 `handleException` 메서드는 `createPhotoUploadJob` 메서드에서 사용되고 있어요.
그렇다면 사진 업로드 시의 `Exception`에 대한 처리를 해주는 부분인데, 
단순히 `handleException`이라는 네이밍이 잘 와닿지 않았어요!
그리고 "필수 값을 모두 입력해 주세요" 라는 내용이 기본 파라미터 값으로 들어가있는 것이 로직의 동작과 맞지 않아 어색하다고 느껴져요.

메서드 명을 변경하고, 기본 파라미터 값에 들어갈 에러 메시지도 함께 바꾸면 좋을 것 같아요~
```suggestion
        private fun handleUpdatePhotoException(
            e: Throwable = IllegalArgumentException(),
            errorMessage: String = IMAGE_UPLOAD_ERROR_MESSAGE,
        ) {
        
        // ...
        private const val IMAGE_UPLOAD_ERROR_MESSAGE = "이미지 업로드에 실패했습니다."
```

Reply:
handleException과 handleUpdatePhotoException를 분리하였습니다!